gapis/atom/group_test.go

```diff
@@ -45,9 +45,9 @@ var tree = `Group 'root' [0..1099]
  │                └─ [0..99] ───── Atoms [500..599]
  └─ [303..702] ── Atoms [600..999]`
 
-func buildTestGroup() Group {
+func buildTestGroup(end uint64) Group {
```

Comment:
Übernit: I was wondering why everything was off by `100`. `end` seems like it should be `root.Range.End`, not the last `atomID-1`. Perhaps use `end` for end range, and `end-100` for the last `atomID-1`?